config/application.rb

```diff
@@ -25,5 +25,18 @@ class Application < Rails::Application
         path.start_with?(Rails.root.join('spec/javascripts').to_s)
       end
     end
+
+    config.middleware.insert_before 0, 'Rack::Cors' do
+      allow do
+        origins '*'
```

Comment:
Luckily, that's what the rack-cors gem does (see the specs, they echo back the Origin header) and the middleware is "silent" without the Origin header as well.